### ISSUE TITLE
Ambiguous statement about QoS-Final-Unit-Indication 

### ISSUE THREAD

**Issue by dcdolson** [lbertz02/rfc4006bis#24]
Regarding this paragraph:
```
   If both Final-Unit-Indication AVP and QoS-Final-Unit-Indication AVP
   exist in the Credit-Control-Answer message, a credit control client
   which supports the QoS-Final-Unit-Indication AVP SHOULD follow the
   directives included in it.
```
Is it supposed to additionally mean that the directives in Final-Unit-Indication should not be followed?

**Comment by yuvalif** [comment]
actually, i would like to leave as is. here is the usecase:
CC server want to comply both with RFC4006 and RFC4006bis clients. So, it send both AVPs.
an RFC4006 will just ignore the new AVP and will follow the Final-Unit-Indication AVP.
A new RFC4006bis client should just follow the new Final-Unit-Indication, and does not need to look into the old one. If you ask them not to follow what the old one says, people may interpret that as a need to verify that they don't contradics, or verify its validity in other ways - which is unnecessary.

**Comment by dcdolson** [comment]
I think you intend an rfc4006bis client SHOULD (MUST?) ignore Final-Unit-Indication and SHOULD (MUST?) implement the directives in QoS-Final-Unit-Indication. Also, the server SHOULD send both AVPs to support RFC4006 clients.

**Comment by yuvalif** [comment]
how about:

>  If both Final-Unit-Indication AVP and QoS-Final-Unit-Indication AVP
   exist in the Credit-Control-Answer message, a credit control client
   which supports the QoS-Final-Unit-Indication AVP SHOULD follow the
   directives included in it, **and SHOULD ignore the Final-Unit-Indication AVP.**

regarding the server side behavior, this is already mentioned in page 87:

>  If the type of Final-Unit-Action AVP is set TERMINATE, or set to
   RESTRICT_ACCESS and the action required is allow only traffic that
   could be classified using an IPFilterRule, or set to REDIRECT of a
   type which is one of the types in the Redirect-Address-Type AVP, then
   the credit-control server SHOULD send the information in the Final-
   Unit-Indication AVP, in addition to or instead of the QoS-Final-Unit-
   Indication AVP.  This is in order to preserve backward compatibility
   with credit-control clients that support only [RFC4006].

**Comment by dcdolson** [comment]
Fixed in commit 7daf935e1be6ddedffb4e9a90e86299f2d0a5989
